pygmt/datasets/earth_age.py

```diff
@@ -44,9 +44,8 @@ def load_earth_age(resolution="01d", region=None, registration=None):
 
     registration : str
         Grid registration type. Either ``"pixel"`` for pixel registration or
-        ``"gridline"`` for gridline registration. Default is ``None``, where
-        a pixel-registered grid is returned unless only the
-        gridline-registered grid is available.
+        ``"gridline"`` for gridline registration. Default is ``"gridline"``
+        when available.
```

Comment:
Looking at https://www.generic-mapping-tools.org/remote-datasets/earth-age.html, it seems like gridline `is` the default for all resolutions anyway? Not sure what the behaviour was for GMT 6.3 though.

```suggestion
        ``"gridline"`` for gridline registration. Default is ``"gridline"``.
```